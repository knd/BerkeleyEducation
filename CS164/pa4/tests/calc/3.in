1 1 / 2 / m / s