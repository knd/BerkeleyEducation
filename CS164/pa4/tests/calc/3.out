[(0, 1, 'Expr -> Fraction'),
 (0, 1, 'Fraction -> Number'),
 (0, 1, 'Number -> [0-9]+'),
 (0, 4, 'Expr -> Fraction'),
 (0, 4, 'Fraction -> Number Number \\/ Number'),
 (0, 6, 'Expr -> Expr \\/ Expr'),
 (0, 8, 'Expr -> Expr \\/ Expr'),
 (1, 2, 'Number -> [0-9]+'),
 (3, 4, 'Number -> [0-9]+'),
 (5, 6, 'Expr -> Unit'),
 (5, 6, 'Unit -> m'),
 (5, 8, 'Expr -> Expr \\/ Expr'),
 (7, 8, 'Expr -> Unit'),
 (7, 8, 'Unit -> s')]
Yes
